autoapi/documenters.py

```diff
@@ -50,6 +62,9 @@ def import_object(self):
                 self._method_parent = parent
                 return True
 
+        # If we get here, the object was not found. Emit a warning as autodoc does.
+        LOGGER.warning("Failed to import %s '%s' [autoapi.import]", self.directivetype, self.fullname, type="autoapi", subtype="import")
+        self.env.note_reread()
```

Comment:
What is this line for?

Reply:
no idea how this came in. I'll delete it.